delfin/drivers/dell_emc/unity/rest_handler.py

```diff
@@ -81,44 +83,47 @@ def login(self):
             LOG.error("Login error: %s", six.text_type(e))
             raise e
 
-    def call_with_token(self, url, data, method):
+    def call_with_token(self, url, data, method, calltimeout):
         auth_key = None
         if self.session:
             auth_key = self.session.headers.get(RestHandler.AUTH_KEY, None)
             if auth_key:
                 self.session.headers[RestHandler.AUTH_KEY] \
                     = cryptor.decode(auth_key)
-        res = self.do_call(url, data, method)
+        res = self.do_call(url, data, method, calltimeout)
         if auth_key:
             self.session.headers[RestHandler.AUTH_KEY] = auth_key
         return res
 
     def logout(self):
         try:
             if self.san_address:
-                self.call(RestHandler.REST_LOGOUT_URL, method='POST')
+                self.call(RestHandler.REST_LOGOUT_URL,
+                          consts.DEFAULT_TIMEOUT,
+                          data={}, method='POST')
             if self.session:
                 self.session.close()
         except Exception as e:
             err_msg = "Logout error: %s" % (six.text_type(e))
             LOG.error(err_msg)
             raise e
 
-    def get_rest_info(self, url, data=None, method='GET'):
+    def get_rest_info(self, url, data=None, method='GET',
+                      calltimeout=consts.DEFAULT_TIMEOUT):
         result_json = None
-        res = self.call(url, data, method)
+        res = self.call(url, calltimeout, data, method)
         if res.status_code == 200:
             result_json = res.json()
         return result_json
 
-    def call(self, url, data=None, method=None):
+    def call(self, url, calltimeout, data=None, method=None):
```

Comment:
Since the parameters of call() is the same as the parameters of call_with_token(), these parameters should be in the same order.

Reply:
done